### PR TITLE
 Change housing type sliders to shares instead of number of units

### PR DESCRIPTION
**Pull request by marliekeverweij**
Screen Shot 2019-06-24 at 16 35 19

- This adjustment simplified the housing type sliders; these now only update the area-file, the useful heat demand in the nodes and insulation costs
- However, the total number of houses input statement now updates everything else (see [input statement](https://github.com/quintel/etsource/blob/d7ee58d3a432546e46b4cf1e3ecffe8ab8db6ff7/inputs/demand/households/households_misc/households_number_of_residences.ad) for more information)

Fixes https://github.com/quintel/etmodel/issues/3024

I have send this PR to the three of you, because I think it's good to test it well, but maybe not everybody! So if you have time :) I did test it locally already!

@redekok will create the scenario migration for these sliders

(Related issue: https://github.com/quintel/etsource/issues/2062)

### CODE REVIEW

File: inputs/demand/households/households_misc/households_number_of_residences.ad [hunk 1]
```diff
@@ -35,10 +34,11 @@
...
     )
   )
+
 - priority = 0
-- max_value_gql = present:AREA(number_of_residences)
-- min_value = 0.0
-- start_value_gql = present:AREA(number_of_semi_detached_houses)
+- max_value_gql = present:AREA(number_of_residences) * 1.0e-06 * 2
+- min_value_gql = present:AREA(number_of_residences) * 1.0e-06 * 0.5
+- start_value_gql = present:AREA(number_of_residences) * 1.0e-06
```
Comment (redekok) [inline inputs/demand/households/households_misc/households_number_of_residences.ad]: I'm not sure if we should express the number of residences in millions. I think this works on country-level but I'm not sure if it works for local scenarios (provinces, municipalities, etc.) as well. Maybe we could express the number in thousands?

File: inputs/demand/households/households_misc/households_number_of_residences.ad [hunk 2]
```diff
@@ -35,10 +34,11 @@
       )
     )
   )
+
 - priority = 0
-- max_value_gql = present:AREA(number_of_residences)
-- min_value = 0.0
-- start_value_gql = present:AREA(number_of_semi_detached_houses)
+- max_value_gql = present:AREA(number_of_residences) * 1.0e-06 * 2
```
[the remark this answers is not in the document]
Comment (antw) [reply inputs/demand/households/households_misc/households_number_of_residences.ad]: > I can help think about this if desired!

```
- max_value_gql = present:AREA(number_of_residences) / MILLIONS * 2
```

File: inputs/demand/households/households_misc/households_number_of_residences.ad (hunk 1 above)
Comment (antw) [reply inputs/demand/households/households_misc/households_number_of_residences.ad]: If the unit is `mln`, I believe ETModel will scale it appropriately.

Comment (marliekeverweij) [reply inputs/demand/households/households_misc/households_number_of_residences.ad]: Yes it works for me locally! :)

[Screen Shot 2019-06-27 at 14 29 57](https://user-images.githubusercontent.com/32833996/60266298-1265b200-98e8-11e9-9f9b-f0348ca4ef82.png)

File: inputs/demand/households/households_misc/households_number_of_residences.ad (hunk 2 above)
[the remark this answers is not in the document]
Comment (marliekeverweij) [reply inputs/demand/households/households_misc/households_number_of_residences.ad]: See https://github.com/quintel/etsource/pull/2063/commits/69fac0e2d0e5dc3de5a5e9e23578a596c7bee9f3

**Comment by antw** [comment]
> While testing, I found out that the new inputs aren't balancing yet. 

Do the sliders sum correctly to 100% in a blank scenario? Judging by the inputs, it seems like they should (as long as `AREA(number_of_residences)` equals `SUM(AREA(number_of_apartments), AREA(number_of_corner_houses), ...)`.

I would expect that ETModel would then ensure that the group balances correctly; do you get an error back from ETEngine?

**Comment by redekok** [comment]
> I would expect that ETModel would then ensure that the group balances correctly; do you get an error back from ETEngine?

False alarm! The share groups weren't defined correctly in my local admin of ETModel. The inputs are balancing well now.

**Pull request merged**